ios/Rainbow.xcodeproj/project.pbxproj

```diff
@@ -759,7 +759,7 @@
 					"$(PROJECT_DIR)",
 				);
 				LLVM_LTO = YES;
-				MARKETING_VERSION = 1.5.1;
+				MARKETING_VERSION = 1.5.11;
```

Comment:
What happened to 1.5.2?

Reply:
i figured follow the pattern we did for 1.2.xx, can for that instead

Reply:
That should happen after 1.5.9

1.5.10
1.5.11

Etc